packages/composables/vitest.config.ts

```diff
@@ -7,14 +7,20 @@ export default defineConfig({
     coverage: {
       enabled: true,
       include: ["src"],
-      // "100": true, // TODO: our goal ;)
       thresholds: {
-        statements: 97,
-        branches: 90,
-        functions: 88.4,
-        lines: 97,
+        statements: 100,
+        branches: 100,
+        functions: 100,
+        lines: 100,
       },
-      exclude: ["**/devtools/**"],
+      exclude: [
+        "**/devtools/**",
+        "**/types/**",
+        "**/src/*.test.ts",
+        "**/src/*.spec.ts",
+        // After refactoring should be removed
```

Comment:
link to GH issue so it's known when resolved when someone will see this TODO in future